client/src/stories/index.js

```diff
@@ -1,19 +1,28 @@
-import React from "react";
-import { storiesOf } from "@storybook/react";
-import App from "./../containers/App/App";
-import { Provider } from "react-redux";
-import configureStore from "../store/configureStore";
-import { BrowserRouter as Router } from "react-router-dom";
-
-
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import App from './../containers/App/App';
+import { Provider } from 'react-redux';
+import configureStore from '../store/configureStore';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { withInfo } from '@storybook/addon-info';
 
 const store = configureStore();
-storiesOf("Authentication", module).add("App", () => 
-  <Provider store={store}>
-    <Router>
-      <App />
-    </Router>
-  </Provider>
+storiesOf('Authentication', module).add(
+	'App',
```

Comment:
what stories would we write for the app component?

Reply:
App story represent App component so here we are making story of app component. It make more sense when write stories order then app component. To make this story clear we can change name from App to home.